### ISSUE TITLE
Adjustments to the `revoked` property

### ISSUE THREAD

**Issue by OR13** [w3c/vc-data-integrity#182]
Multikey defines `revoked` but `JsonWebKey` does not.

```
"revoked": {
  "@id": "https://w3id.org/security#revoked",
  "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
},
```

On https://github.com/w3c/vc-jose-cose/pull/144#discussion_r1305826152 @TallTed asks if this field should include TZ.

If we make such a change, we would want to ensure it applies to all Verification Method classes, and was documented the same in both specs.

**Comment by msporny** [comment]
> we would want to ensure it applies to all Verification Method classes, and was documented the same in both specs.

Agree.

For the time being, I'll just bring both VM classes inline w/ each other.

**Comment by msporny** [comment]
PR #184 has been raised to address this issue. This issue will be closed once PR #184 is merged.

**Comment by msporny** [comment]
PR #184 has been merged, closing.
